test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -129,6 +131,59 @@ def func(x, b=torch.tensor(1.0)):
             func, (tensor_x,), input_kwargs={"b": torch.tensor(5.0)}
         )
 
+    @pytorch_test_common.skip_dynamic_fx_test(
+        "sympy operation tests doesn't need dynamic shape"
+    )
+    def test_sympy_operatons_return_numeric(self):
+        def func(x, y):
+            return (
+                # torch.tensor([operator.add(x.item(), y.item())]),
+                # torch.tensor([operator.sub(x.item(), y.item())]),
+                # torch.tensor([operator.mul(x.item(), y.item())]),
+                # torch.tensor([operator.truediv(x.item(), y.item())]),
+                # torch.tensor([operator.floordiv(x.item(), y.item())]),
+                # torch.tensor([operator.pow(x.item(), y.item())]),
+                # torch.tensor([operator.abs(x.item())]),
+                # torch.tensor([operator.neg(x.item())]),
```

Comment:
why all these comments? are they from previous testing?

either delete them or add an additional explanation for them